streaming/kafka-confluent/terraform/gke-replicator/README.md

```diff
@@ -0,0 +1,26 @@
+# Terraform to provision GKE Standard
+
+## Prerequisites and Assumptions
+* Done initialization of the project and gcloud CLI following the instructions in `{ROOT}/README.md`
+* VPC network, refer to `gke` folder for the details
+
+## Usage
+```
+export GOOGLE_OAUTH_ACCESS_TOKEN=$(gcloud auth print-access-token)
+export PROJECT_ID="your project"
+export REGION="us-central1"
+export CLUSTER_PREFIX="strimzi"
```

Comment:
**Question:** Should this be
```suggestion
export CLUSTER_PREFIX="confluent"
```
(but see my other comment about reducing Terraform duplication)